Configuration/Generator/python/EtaBToJpsiJpsi_forSTEAM_14TeV_TuneCP5_cfi.py

```diff
@@ -0,0 +1,76 @@
+# For BPH trigger study only
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+from GeneratorInterface.EvtGenInterface.EvtGenSetting_cff import *
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
+  maxEventsToPrint = cms.untracked.int32(1),
+  pythiaPylistVerbosity = cms.untracked.int32(1),
+  filterEfficiency = cms.untracked.double(0.4045),
+  pythiaHepMCVerbosity = cms.untracked.bool(False),
+  comEnergy = cms.double(14000.),
+  PythiaParameters = cms.PSet(
+    pythia8CommonSettingsBlock,
+    pythia8CP5SettingsBlock,
+    processParameters = cms.vstring(
+      'Higgs:useBSM = on',
+      'HiggsBSM:gg2H2 = on',
+      'HiggsH2:coup2d = 10.0',
+      'HiggsH2:coup2u = 10.0',
+      'HiggsH2:coup2Z = 0.0',
+      'HiggsH2:coup2W = 0.0',
+      'HiggsA3:coup2H2Z = 0.0',
+      'HiggsH2:coup2A3A3 = 0.0',
+      'HiggsH2:coup2H1H1 = 0.0',
+      '443:onMode = off',
+      '443:onIfMatch 13 -13',
+      ############# Fix to mass of etaB (9.4 GeV) #############
+      '35:mMin = 0',
+      '35:mMax = 50.0',
+      '35:m0 = 9.4',
+      '35:mWidth = 0.00',
+      '35:addChannel 1 1.00 100 443 443',
+      '35:onMode = off',
+      '35:onIfMatch 443 443' #Jpsi Jpsi decay
+    ),
+      # This is a vector of ParameterSet names to be read, in this order
+    parameterSets = cms.vstring(
+      'pythia8CommonSettings',
+      'pythia8CP5Settings',
+      'processParameters'
+    )
+  )
+)
+
+generator.PythiaParameters.processParameters.extend(EvtGenExtraParticles)
+
+etafilter = cms.EDFilter("PythiaFilter",
```

Comment:
why is this called etafilter?

Reply:
I did as in previous version of the fragment available in CMSSW (https://github.com/cms-sw/cmssw/blob/02d443bd49e0f29381709421b4a812496f9b8d1a/Configuration/Generator/python/EtaBToJpsiJpsi_forSTEAM_TuneCUEP8M1_13TeV_cfi.py#L44)

The BSM Higgs (pdgid=35) with mass forced to the same as etaB is used instead of an etaB.